packages/app-builder-lib/templates/linux/after-install.tpl

```diff
@@ -6,5 +6,10 @@ ln -sf '/opt/${sanitizedProductName}/${executable}' '/usr/bin/${executable}'
 # SUID chrome-sandbox for Electron 5+
 chmod 4755 '/opt/${sanitizedProductName}/chrome-sandbox' || true
 
-update-mime-database /usr/share/mime || true
-update-desktop-database /usr/share/applications || true
+if hash update-mime-database 2>/dev/null; then
+    update-mime-database /usr/share/mime
```

Comment:
Is there ever a chance this could fail? Would it be worth keeping the `|| true` for both commands to play it safe?

Reply:
`update-mime-database` can execute correctly. `|| true` [used to ignore error](https://unix.stackexchange.com/questions/325705/why-is-pattern-command-true-useful/325727) , but error message is still displayed. `if hash update-mime-database 2>/dev/null` check the command before execution, no error message is displayed. I have installed vscode under Ubuntu 20.04(GNOME), works fine. VScode has a large number of users, so there should be no problem.

`|| true` and `if hash update-mime-database 2>/dev/null` both used to ignore error, keep bash script continue execution.  Since it doesn't matter if there is an error, there is no need to display an error message.

Of course, the safest thing is to keep `update-mime-database /usr/share/mime || true` and only change `update-desktop-database`.